cypress/support/step_definitions/dashboard/there_is_a_list_with_all_states.js

```diff
@@ -1,6 +1,15 @@
+/* eslint-disable */
 import { rand, randCompanyName, randFullName } from "@ngneat/falso";
 
 Given("A lawyers list exists for Eurasia", () => {
+  createListForService("lawyers");
+});
+
+Given("A funeral directors list exists for Eurasia", () => {
+  createListForService("funeralDirectors");
+});
+
+function createListForService(service) {
```

Comment:
Refactor to
```
```suggestion
Given("A {string} list exists for Eurasia", (providerType) => {
  createListForService(providerType);
});

function createListForService(service) {
```
```